src/masternodes/mn_checks.cpp

```diff
@@ -405,7 +414,14 @@ Res ApplyUpdateTokenAnyTx(CCustomCSView & mnview, CCoinsViewCache const & coins,
         return Res::Err("%s: %s", base, "tx must have at least one input from token owner");
     }
 
-    auto res = mnview.UpdateToken(token.creationTx, newToken, false);
+    if (!isFoundersToken && updateFlags & static_cast<uint8_t>(TokenUpdateFlags::NewOwner)) {
```

Comment:
Can put `updateFlags & static_cast<uint8_t>(TokenUpdateFlags::NewOwner)` inside a (), it is easier to read.

Reply:
Wil do. There's a couple of locations to update.